roles/icpc_host_backup/files/iptables.rules.new

```diff
@@ -0,0 +1,143 @@
+*filter
+:INPUT ACCEPT [0:0]
+:FORWARD DROP [0:0]
+:OUTPUT ACCEPT [0:0]
+
+# LOGGING / DEBUGGING
+-N accept-in-log
+-A accept-in-log -j LOG --log-level 4 --log-prefix "accept-in-log:"
+-A accept-in-log -j ACCEPT
+-N accept-out-log
+-A accept-out-log -j LOG --log-level 4 --log-prefix "accept-out-log:"
+-A accept-out-log -j ACCEPT
+-N drop-n-log
+-A drop-n-log -j LOG -m limit --limit 5/m --limit-burst 7 --log-level 4 --log-prefix "DENIED:"
+-A drop-n-log -j DROP
+
+# internal interface
+-A INPUT -i lo -j ACCEPT
+-A OUTPUT -o lo -j ACCEPT
+
+# open up server to server communication
+-A OUTPUT -d 10.3.3.208/29 -j ACCEPT
+-A INPUT -s 10.3.3.208/29 -j ACCEPT
+
+# ldap inbound
```

Comment:
Given that ldap is a group we can loop over that group and for each host pick the IP.

Reply:
actually i can drop the ldap bit, as we aren't doing ldap anymore.

Reply:
or changed them behind a ldap variable in case we ever go back to using ldap.